client/src/pages/ThankYouPage.jsx

```diff
@@ -0,0 +1,53 @@
+import "../styles/ThankYouPage.css";
+
+function ThankYouPage() {
+  return (
+    <>
+      <h1>Remerciements !</h1>
+      <section className="thankyou-page">
+        <h2>
+          Nous voulons remercier tous nos amis utilisateurs qui comme nous adore
+          l'art de rue.
+        </h2>
+        <p>
+          Toute l'équipe, Adrien, Alice, Christophe et Kevin sont fiers de vous

```

Comment:
spelling mistake "toute l'équipe... sont fiers" should be "toute l'équipe, composée de ..., est fière"
but as in my previous comment it is not us but our client who thanks the users

Reply:
true